File: python/stack_reference.py
def main():
    stack = []

    stack.append(1)
    stack.append(2)
    stack.append(3)
    stack.pop()
    print(stack)

    print(stack[-1])       # peek
    print(len(stack) == 0)

if __name__ == "__main__":
    main()

File: typescript/stack_reference.ts
export function main() {
    let stack: number[] = [];

    // Push
    stack.push(1);
    stack.push(2);

    // Peek, Pop
    console.log(stack[stack.length - 1]);
    console.log(stack.pop());
    console.log(stack.length === 0);
}

main();